ledger/ledger-utxo/src/test/java/net/corda/v5/ledger/utxo/AbstractMockTestHarness.java

```diff
@@ -1,10 +1,14 @@
 package net.corda.v5.ledger.utxo;
 
+import net.corda.v5.application.crypto.DigitalSignatureAndMetadata;
+import net.corda.v5.application.crypto.DigitalSignatureMetadata;
+import net.corda.v5.application.serialization.SerializationService;
 import net.corda.v5.base.types.MemberX500Name;
 import net.corda.v5.cipher.suite.DigestService;
+import net.corda.v5.crypto.DigitalSignature;
 import net.corda.v5.crypto.SecureHash;
 import net.corda.v5.ledger.common.transaction.Party;
-import net.corda.v5.ledger.utxo.transaction.UtxoLedgerTransaction;
+import net.corda.v5.ledger.utxo.transaction.*;
```

Comment:
AFAIK `*` imports are accepted.

Reply:
Hmm. I missed only the point here...
Correctly:

AFAIK * imports are *NOT* accepted.